src/app/app.module.ts

```diff
@@ -110,7 +110,7 @@ import { realmProvider }                 from './shared/realm-token.provider';
 import { ConfigStore }               from './base/config.store';
 import { ErrorService }              from './layout/error/error.service';
 import { ProfileService }            from './profile/profile.service';
-import { SpaceWizardModule }         from './space-wizard/space-wizard.module';
+import { SpaceWizardModule }         from './space/wizard/space-wizard.module';
```

Comment:
Do we still need the module named "space-wizard"? I'm good with it, just wanted to confirm.